second-analysis-steps/managing-files-with-ganga.md

```diff
@@ -209,26 +209,40 @@ it's not currently available at.
 df.replicate('RAL-USER')
 ```
 
-{% callout "Automating replication to CERN" %}
+{% callout "Accessing DiracFiles remotely" %}
 
-If you have a job with subjobs, you can automate this to replicate all output 
-files to CERN, so that you can run your analysis directly on the files on 
-EOS.
+A DiracFile can be accessed remotely if you have a valid grid proxy.
+
+You need to find the `accessURL` (also known as the Physical File Name or `PFN`)
+which tells you the location of your file. The `LFN` (Logical File Name) is the 
+record of the file in the Dirac database.
+
+To get the `accessURL` you can use the `LHCbDirac` command line option with an LFN:
+
+```
+lb-dirac dirac-dms-accessURL /path/to/some/LFN.root
```

Comment:
The command is wrong, should be `dirac-dms-lfn-accessURL`

Reply:
Good spot